notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>Where did you call <code>json.loads()</code>? Can you show how to parse the prediction using <code>json.loads()</code>  to show just the predicted classes, after you show the entire prediction?</p><p> </p><p>Change section title to "Download and parse the sample prediction" or something similar. That will make it clearer what the blob is, and what it's for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Yes, I call it in the next section. I move parse it using json.loads() method to the next header.</p><p>The </p>